Forge/src/main/java/me/ferdz/placeableitems/block/PlaceableItemsBlock.java

```diff
@@ -52,28 +53,19 @@ public PlaceableItemsBlock() {
         this.components = new ArrayList<>();
     }
 
-    public PlaceableItemsBlock register(String name) {
+    public PlaceableItemsBlock register(String name, IForgeRegistry<Block> registry) {
         this.setRegistryName(PlaceableItems.MODID, name);
-        // FIXME: This should use the event instead of accessing the registry directly
-        GameRegistry.findRegistry(Block.class).register(this);
+        registry.register(this);
```

Comment:
I was hoping to avoid this, but I'm guessing this is unavoidable :(

Reply:
It is, though it's also the best way to do it. `GameRegistry.findRegistry()` is cheap, it's an O(1) operation, but avoiding it and instead passing a value we have available is definitely preferred.

Reply:
Nope, let's keep it as passing the registry